src/blueapi/service/runner.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import signal
+from collections.abc import Callable, Iterable
+from multiprocessing import Pool, set_start_method
+from multiprocessing.pool import Pool as PoolClass
+from typing import Any
+
+from blueapi.config import ApplicationConfig
+from blueapi.service.interface import start_worker, stop_worker
+from blueapi.service.model import (
+    EnvironmentResponse,
+)
+
+# The default multiprocessing start method is fork
+set_start_method("spawn", force=True)
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _init_worker():
+    # Replace sigint to allow subprocess to be terminated
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+
+
+class Runner:
```

Comment:
I would prefer this have a longer, more descriptive name, maybe `WorkerProcessDispatcher`?

Reply:
Yeah I did try this although it is used a lot in already very long function definitions in the rest app main.py and made them a bit more unwieldy, or that might have just been me. Could go with `WorkerDispatcher` for now?

Reply:
Yes please

Reply:
Done